master/reference/calicoctl/resources/ippool.md

```diff
@@ -42,4 +43,10 @@ spec:
 
 | Field    | Description                 | Accepted Values | Schema  | Default    |
 |----------|-----------------------------|--------------|---------|------------|
-| enabled   | When set to true, ipip encapsulation will be used. | true, false | boolean | true
+| enabled  | When set to true, ipip encapsulation will be used. | true, false | boolean | true |
+| mode     | The IPIP mode defining when IPIP will be used.     | always, cross-subnet | string | always |
+
+(*) Routing of packets using IP in IP will be used when the destination IP address
+is in an IP Pool that has IPIP enabled.  In addition, if the `mode` is set to `cross-subnet`,
+Calico will only route using IP in IP if the IP address of the destination node is different
```

Comment:
"different to the subnet of the originating node." --(could be simplified to)--> "in a different subnet"